File: Year 3/React Native/Simple SMHI Weather Warning application/my-second-project/models/Event.ts
import { MhoClassification } from './MhoClassification';

class Event {
    en: string;
    sv: string;
    code: string;
    mhoClassification: MhoClassification;

    constructor(en: string = '', sv: string = '', code: string = '', mhoClassification?: MhoClassification) {
        this.en = en;
        this.sv = sv;
        this.code = code;
        this.mhoClassification = mhoClassification || new MhoClassification();
    }

    getEn(): string | null {
        return this.en;
    }

    setEn(en: string): void {
        this.en = en;
    }

    getSv(): string | null {
        return this.sv;
    }

    setSv(sv: string): void {
        this.sv = sv;
    }

    getCode(): string | null {
        return this.code;
    }

    setCode(code: string): void {
        this.code = code;
    }

    getMhoClassification():MhoClassification | null {
        return this.mhoClassification;
    }

    setMhoClassification(mhoClassification: MhoClassification): void {
        this.mhoClassification = mhoClassification;
    }
}
export {Event};

File: Year 3/Android Java/Simple SMHI Warning App JavaVersion/GeoWarning/app/src/main/java/com/example/geowarning/model/Event.java
package com.example.geowarning.model;

public class Event {
    private String en;
    private String sv;
    private String code;
    private MhoClassification mhoClassification;

    public String getEn() {
        return en;
    }

    public void setEn(String en) {
        this.en = en;
    }

    public String getSv() {
        return sv;
    }

    public void setSv(String sv) {
        this.sv = sv;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public MhoClassification getMhoClassification() {
        return mhoClassification;
    }

    public void setMhoClassification(MhoClassification mhoClassification) {
        this.mhoClassification = mhoClassification;
    }
}
